src/smart-escrow/SmartEscrow.sol

```diff
@@ -169,6 +196,16 @@ contract SmartEscrow is AccessControlDefaultAdminRules {
         }
     }
 
+    /// @notice Allow withdrawal of remaining tokens to benefactor address if contract is terminated.
+    /// @notice Emits a {Transfer} event.
+    function withdrawUnvestedTokens() public onlyRole(DEFAULT_ADMIN_ROLE) {
+        if (!contractTerminated) revert ContractIsNotTerminated();
+        uint256 amount = OP_TOKEN.balanceOf(address(this));
+        if (amount > 0) {
+            SafeERC20.safeTransfer(OP_TOKEN, benefactor, amount);
```

Comment:
There was a suggestion to include extra events here and in release in addition to the Transfer event - thoughts?

Reply:
pretty low cost, not opposed